src/System.Private.Uri/src/System/Uri.cs

```diff
@@ -447,16 +447,16 @@ protected void GetObjectData(SerializationInfo serializationInfo, StreamingConte
             }
         }
 
-        private void CreateUri(Uri baseUri, string relativeUri, bool dontEscape)
+        private void CreateUri(Uri baseUri, string? relativeUri, bool dontEscape)
```

Comment:
There are a couple of constructors that call into this API that take `relativeUri` those should also be nullable.

Reply:
Fixed